app/src/main/java/com/x8bit/bitwarden/ui/platform/feature/tiles/GeneratorTileService.kt

```diff
@@ -0,0 +1,53 @@
+package com.x8bit.bitwarden.ui.platform.feature.tiles
+
+import android.annotation.SuppressLint
+import android.os.Build
+import android.service.quicksettings.TileService
+import com.x8bit.bitwarden.ui.platform.manager.intent.IntentManager
+import com.x8bit.bitwarden.ui.platform.manager.intent.IntentManagerImpl
+import kotlinx.coroutines.Runnable
+
+/**
+ * A service for handling the Password Generator quick settings tile.
+ */
+class GeneratorTileService : TileService() {
```

Comment:
Annotate the class with:
- `@AndroidEntryPoint` so that `IntentManager` can be injected.
- `@Keep` so that it isn't stripped out or renamed during minification.
- `@OmitFromCoverage` since we don't test service classes.

Reply:
For stuff like this, we do omit this file from coverage but all the logic should be offloaded to a `GeneratorTileProcessor` that can be tested.